src/switch_core_media.c

```diff
@@ -7486,6 +7547,9 @@ SWITCH_DECLARE(uint8_t) switch_core_media_negotiate_sdp(switch_core_session_t *s
 			if (!smh->mparams->rtcp_audio_interval_msec) {
 				smh->mparams->rtcp_audio_interval_msec = SWITCH_RTCP_AUDIO_INTERVAL_MSEC;
 			}
+			if (!smh->mparams->partner_rtcp_audio_interval_msec) {
```

Comment:
We shouldn't initialize this right and should be 0 by default to prevent this profile updating the other channel profile when no profile configuration is set.